src/components/Addbook.jsx

```diff
@@ -0,0 +1,25 @@
+import { useState } from 'react';
+
+const Addbook = () => {
+  const [book, setBook] = useState('');
+
+  const handleSubmit = (e) => {
+    e.preventDefault();
+  };
+
+  return (
+    <>
+      <form onSubmit={handleSubmit}>
+        <input
+          type="text"
+          value={book}
+          onChange={(e) => setBook(e.target.value)}
+          placeholder="Add book"
+        />
```

Comment:
- [ ] Kindly add another input field for the author in your form as demanded by project requirements.